config-bootstrapper/build.gradle.kts

```diff
@@ -87,11 +87,11 @@ tasks.register<DockerStopContainer>("stopAttributeServiceContainer") {
 }
 
 tasks.register<DockerPullImage>("pullEntityServiceImage") {
-  image.set("hypertrace/entity-service:0.1.0")
+  image.set("hypertrace/entity-service:latest")
```

Comment:
what?

Reply:
:) Sorry.. this was for my local testing. Changed now.